src/redux/slices/settingsSlice/index.ts

```diff
@@ -228,6 +229,13 @@ export const unlockApp = createAsyncThunk<
   AsyncThunkWithTypes
 >('settings/unlockApp', async (payload, thunkAPI) => {
   try {
+    if (!Config.TRACE_ID) {
+      const { setGlobalError } = payload
+      const errorMessage = 'Trace Id is not valid'
+      setGlobalError?.(errorMessage)
```

Comment:
Why is this optional? As far as I understand it's required

Reply:
Done!